DEV_SETUP_MAC.md

```diff
@@ -175,6 +175,22 @@ NOTE: Make sure that `/path/to` is replaced with the actual path!
 
 After this you can open a new terminal window and run elasticsearch with `elasticsearch`.
 
+#### Install Elasticsearch plugins
+
+Now that you have Elasticsearch running you will need to install the necessary plugins:
+
+**Elasticsearch v2**
+```
+$ <es home>/bin/plugin install analysis-phonetic
```

Comment:
Know this is the technically correct way, but would it make more sense to have just `plugin install analysis-phonetic` because directions above already have us add elasticsearch to path?